genai-perf/genai_perf/inputs/retrievers/synthetic_prompt_generator.py

```diff
@@ -110,9 +131,10 @@ def word_generator():
         prompt += final_line
 
         # Final tweaks
-        diff = requested_prompt_tokens - get_token_length(prompt)
-        for _ in range(diff):
-            prompt = "hi " + prompt
+        for _ in range(2):
+            diff = requested_prompt_tokens - get_token_length(prompt)
+            for _ in range(diff):
+                prompt = "hi " + prompt
```

Comment:
This should actually always be the case to ensure consistent number of tokens in input I think